### PR TITLE
electron-fiddle: Add version 0.26.0

### LINKED ISSUE #6578: Add electron-fiddle
https://www.electronjs.org/fiddle

Electron Fiddle is a very useful app for Electron developers

### PR DESCRIPTION
**Pull request by minonl**
- Closes #6578

### CODE REVIEW

File: bucket/electron-fiddle.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "0.26.0",
+    "description": "The easiest way to get started with Electron",
+    "homepage": "https://www.electronjs.org/fiddle",
+    "license": "MIT",
+    "url": "https://github.com/electron/fiddle/releases/download/v0.26.0/electron-fiddle-0.26.0-full.nupkg#/dl.7z",
+    "hash": "sha1:EFE54F507C1832BD740BB0BEA59631E009389853",
+    "extract_dir": "lib\\net45",
+    "bin": "electron-fiddle.exe",
```
Comment (Ash258) [inline bucket/electron-fiddle.json:9]:
```suggestion
    "suggest": {
        "npm": "npm",
        "yarn": "yarn"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/electron/fiddle/releases/download/v0.26.0/electron-fiddle-0.26.0-win32-x64-setup.exe#/dl.7z",
            "hash": "3ff1cc334b82c94b71e1eb3addccb7ac6454f6970fb42ec6ce05f1fd401c1af6"
        },
        "32bit": {
            "url": "https://github.com/electron/fiddle/releases/download/v0.26.0/electron-fiddle-0.26.0-win32-ia32-setup.exe#/dl.7z",
            "hash": "494ddaa3c90bbd3b8557214188b496dc66dbb3125f68c3ed6722f9af503adb04"
        }
    },
    "pre_install": [
        "Get-ChildItem \"$dir\\*\" -Exclude 'electron-fiddle-*nupkg' | Remove-Item",
        "Expand-7zipArchive \"$dir\\electron-fiddle-*-full.nupkg\" -ExtractDir 'lib\\net45' -Removal"
    ],
```

File: bucket/electron-fiddle.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "0.26.0",
+    "description": "The easiest way to get started with Electron",
```
Comment (Ash258) [inline bucket/electron-fiddle.json:3]: Marketing text which does not mean anything to user as it does not describe the application.

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/electron-fiddle.json (hunk 2 above)
Comment (minonl) [reply bucket/electron-fiddle.json]: I googled and replaced the description. Also it stay same in [homebrew](https://github.com/minonl/homebrew-cask/blob/040dffb345b5c48688c5707311a8bfbe9fe9e36d/Casks/electron-fiddle.rb#L17)

**Comment by minonl** [comment]
@Ash258 Please execuse my ignorance 😵, I noticed there is still 1 change requested despite all previous changes of https://github.com/lukesampson/scoop-extras/pull/6579#pullrequestreview-724904764  have been commited and marked resolved.
I wonder if there is anything I should do but I didn't. Please let me know if so

**Review by Ash258 — APPROVED** [review]

**Pull request merged**